### PR TITLE
Faqs and accordian updates

### PR DESCRIPTION
**Pull request by Panguino**
## Monday.com Item

Monday Item ID: 7553803806

## Description

Added Faqs page and updated accordian component

## Types of changes

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/components/elements/Accordian/Accordian.module.scss [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
+.accordian {
   position: relative;
+  margin-bottom: 10px;
 }
 .title {
-  background: #434343;
-  padding: 10px 13px;
+  background: var(--color-grey4-grey16);
+  padding: 8px 13px;
   color: #fff;
```
Comment (elusiveuphoria) [inline src/components/elements/Accordian/Accordian.module.scss]: Needs to use light/dark mode color variables.

**Review by elusiveuphoria — CHANGES_REQUESTED** [review]
Headers and Links need light/dark mode colors.

And fix mobile responsiveness; wrap accordion below title content.

File: src/components/elements/Accordian/Accordian.module.scss (hunk 1 above)
Comment (Panguino) [reply src/components/elements/Accordian/Accordian.module.scss]: This is fixed here b6c78755686c093e353962d26512105e1e6c5fd4  new preview links should show updates.

Comment (elusiveuphoria) [reply src/components/elements/Accordian/Accordian.module.scss]: Looks good

**Pull request merged**